core/src/main/resources/jenkins/security/csrf/CSRFAdministrativeMonitor/message_ru.properties

```diff
@@ -0,0 +1,6 @@
+warningMessage=Вы не настроили выдачу CSRF-токенов. Это может быть небезопасно. \
+  Больше информации смотрите на {0}.
+actionMessage=Вы можете изменить текущую настройку в разделе Безопасности: {0}.
```

Comment:
```suggestion
actionMessage=Вы можете изменить текущую настройку в разделе Безопасность: {0}.
```

Reply:
did that and put it in quotes since it's the name, looks better imo